BraveWallet/Crypto/Accounts/Add/AddAccountView.swift

```diff
@@ -114,6 +91,64 @@ struct AddAccountView: View {
     }
   }
 
+  var body: some View {
+    Group {
+      if coin == nil && WalletConstants.supportedCoinTypes.count > 1 {
+        List {
```

Comment:
Can we separate this into a computed property like `addAccountView`? Just for readability

Reply:
fixed in [a2692d6](https://github.com/brave/brave-ios/pull/5651/commits/a2692d6f855ed7bf1e050481d7749772d6fab014)